src/Resources/skeleton/doctrine/broadcast_twig_template.tpl.php

```diff
@@ -0,0 +1,19 @@
+{% block create %}
+<turbo-stream action="append" target="<?= $class_name_plural ?>">
+    <template>
+        <div id="{{ '<?= $class_name ?>_' ~ entity.id }}">#{{ entity.id }} created</div>
+    </template>
+</turbo-stream>
+{% endblock %}
+
+{% block update %}
+<turbo-stream action="update" target="<?= $class_name ?>_{{ entity.id }}">
+    <template>
+        #{{ entity.id }} updated
+    </template>
+</turbo-stream>
+{% endblock %}
+
+{% block remove %}
+<turbo-stream action="remove" target="<?= $class_name ?>_{{ entity.id }}"></turbo-stream>
```

Comment:
should be `id` eveywhere instead of `entity.id`

Reply:
Why not but on the other hand here we're sure that `entity.id` works and this hints the existence of the `entity` variable.

Reply:
For remove, it doesn't work, but for others yes

Reply:
Oh, and id works even if the property is not named id

Reply:
Good catch for remove. (When using maker-bundle the property is always named id).